src/pages/LoginSignup/components/Input.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import './input.scss';
+
+const Input = ({ type, text, value, handleInput, isLogin, isValid }) => {
+  return (
+    <div className="input">
+      <div className="inputWrapper">
+        <input
+          className="signInput"
+          name={type}
+          type={type}
+          placeholder={text}
+          value={value}
+          onChange={handleInput}
+        />
+        {isLogin ? null : isValid ? <ValidCheck /> : <InValidCheck />}
```

Comment:
```suggestion
        {isValid !== undefined && isValid ? <ValidCheck /> : <InValidCheck />}
```

isValid값을 Singup일때만 전달받는다면, 위와 같이 처리 가능해보이네요
그러면 isLogin이란 값을 받을 필요가 없어지고요!

참고해서 반영해보세요!